paasta_tools/kubernetes_tools.py

```diff
@@ -170,6 +170,7 @@
 UWSGI_EXPORTER_POD_NAME = "uwsgi--exporter"
 SIDECAR_CONTAINER_NAMES = [HACHECK_POD_NAME, UWSGI_EXPORTER_POD_NAME]
 KUBERNETES_NAMESPACE = "paasta"
+PAASTA_WORKLOAD_OWNER = "compute_infra_platform_experience"
```

Comment:
it'd be nice to have this come from systempaastaconfig, but it's fine as-is (i.e., we can always enhance this later)

Reply:
I think we can punt on this for now (even if it takes a couple months) :)